src/security/tests/drpc_mocks.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * (C) Copyright 2019 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+ * The Government's rights to use, modify, reproduce, release, perform, display,
+ * or disclose this software are subject to the terms of the Apache License as
+ * provided in Contract No. 8F-30005.
+ * Any reproduction of computer software, computer software documentation, or
+ * portions thereof marked with this legend must also reproduce the markings.
+ */
+
+/**
+ * Mocks of dRPC framework functions
+ */
+
+#ifndef __DAOS_DRPC_MOCKS_H__
+#define __DAOS_DRPC_MOCKS_H__
+
+#include <daos/drpc.h>
+#include "../security.pb-c.h"
+
+/**
+ * drpc_connect mock values
+ */
+extern struct drpc *drpc_connect_return; /* value to be returned */
+extern char drpc_connect_sockaddr[PATH_MAX + 1]; /* saved copy of input */
+
+void mock_drpc_connect_setup(void);
+void mock_drpc_connect_teardown(void);
+
+/* Convenience method to free mocks */
+void free_drpc_connect_return();
```

Comment:
(style)  Bad function definition - void free_drpc_connect_return() should probably be void free_drpc_connect_return(void)